docs/development/setup-recommended.md

```diff
@@ -740,485 +740,44 @@ Next, read the following to learn more about developing for Zulip:
 
 ### Troubleshooting and common errors
 
-Below you'll find a list of common errors and their solutions. Most
-issues are resolved by just provisioning again (by running
-`./tools/provision` (from `/srv/zulip`) inside the Vagrant guest or
-equivalently `vagrant provision` from outside).
```

Comment:
Let's have a shared intro section here before the tabs. In places where we mention Vagrant, we can add "or WSL instance" or something equivalent.